gossip/Cargo.toml

```diff
@@ -61,7 +61,10 @@ static_assertions = { workspace = true }
 thiserror = { workspace = true }
 
 [dev-dependencies]
+bs58 = { workspace = true }
 num_cpus = { workspace = true }
+rand0-7 = { workspace = true }
+rand_chacha0-2 = { workspace = true }
```

Comment:
why do we need `rand_cha_cha` v2 here? Can we not get away with v3?

Reply:
This code will not compile unless I use older versions for both rand and rand_chacha because `Keypair::generate` requires a rand 0.7 `CryptoRng` and rand_chacha 0.3 implements rand 0.8 `CryptoRng` which is not the same as 0.7 version.

Reply:
love dependency hell lol. sounds good!